infra/k8s/kops-weave/s3bucket.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: s3bucket
+  namespace: default
+  labels:
+    k8s-app: s3bucket
+spec:
+  selector:
+    matchLabels:
+      name: s3bucket
+  template:
+    metadata:
+      annotations:
+        cni: "flannel"
+      labels:
+        name: s3bucket
+    spec:
+      containers:
+        - name: s3fs
+          image: nonsens3/s3fs
```

Comment:
Is this a fork you maintain, @nonsense?

Reply:
There is no official version for this one, I have committed here the Dockerfile, but yeah - we should move that to the `ipfs` docker repo.

Reply:
Added a TODO, will move to an issue and we can fix later.